frontend/app-development/layout/PageHeader/LargeNavigationMenu/LargeNavigationMenu.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { type ReactElement } from 'react';
+import classes from './LargeNavigationMenu.module.css';
+import { NavLink, useLocation } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import { StudioPageHeaderButton } from '@studio/components';
+import { Tag } from '@digdir/designsystemet-react';
+import { getRouterRouteByPathname } from 'app-development/utils/headerMenu/headerMenuUtils';
+import { type NavigationMenuItem } from 'app-development/types/HeaderMenu/NavigationMenuItem';
+import { usePageHeaderContext } from 'app-development/contexts/PageHeaderContext';
+
+export type LargeNavigationMenuProps = {
+  menuItems: NavigationMenuItem[];
+};
+
+export const LargeNavigationMenu = ({ menuItems }: LargeNavigationMenuProps): ReactElement => {
+  const { t } = useTranslation();
+  const { variant } = usePageHeaderContext();
+
+  const location = useLocation();
+  const currentRoutePath: string = getRouterRouteByPathname(location.pathname);
+
+  return (
+    <div className={classes.wrapper}>
+      <ul className={classes.menu}>
+        {menuItems.map((menuItem: NavigationMenuItem) => (
+          <li key={menuItem.name}>
+            <StudioPageHeaderButton asChild color='dark' variant={variant}>
+              <NavLink to={menuItem.link}>
+                <span
+                  className={
+                    getRouterRouteByPathname(menuItem.link) === currentRoutePath
+                      ? classes.active
+                      : undefined
+                  }
```

Comment:
What about using the `classnames` package to deal with conditional classnames. What do you think? 😊 

```suggestion
                  className={cn({ classes.active: getRouterRouteByPathname(menuItem.link) === currentRoutePath })        
```

Reply:
I'm getting an error when doing that saying it is not allowed 🤔

Reply:
What kind of error do you get? 🤔

Reply:
![Skjermbilde 2024-09-26 kl  07 27 38](https://github.com/user-attachments/assets/ea338ce7-a6a5-4621-
![Skjermbilde 2024-09-26 kl  07 28 00](https://github.com/user-attachments/assets/fe06b920-09a7-4c57-9262-90126267bc94)
a9fa-d803cd63ab6f)

Reply:
By adding [ ] around the classes.active should do the trick. 🙌 
```ts
className={cn({ [classes.active]: getRouterRouteByPathname(menuItem.link) === currentRoutePath })   
```

Reply:
Brilliant, very nice trick! 🥳